.eslintrc.yml

```diff
@@ -126,6 +126,8 @@ rules:
 # Global scoped method and vars
 globals:
   netlify: true
+  global: true
+  __dirname: true
```

Comment:
@bdougie I add this to prevent these errors!

![Screenshot](https://user-images.githubusercontent.com/61746440/85573749-cb71e000-b653-11ea-822f-d7d086027ba5.png)